### PR TITLE
[external-dns] Added affinity.

### PR DESCRIPTION
**Pull request by michalschott**
**What this PR does / why we need it**:
It adds affinity to external-dns pod.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
N/A

**Special notes for your reviewer**:
None

### CODE REVIEW

**Review by davidkarlsen — CHANGES_REQUESTED** [review]
You need to bump the Chart.yaml version too

File: stable/external-dns/templates/deployment.yaml [hunk 1]
```diff
@@ -141,5 +141,7 @@ spec:
     {{- if .Values.tolerations }}
       tolerations:
 {{ toYaml .Values.tolerations | indent 8 }}
+      affinity:
+{{ toYaml .Values.affinity | indent 8 }}
```
Comment (davidkarlsen) [inline stable/external-dns/templates/deployment.yaml]: should maybe be outside the tolerations conditional?

**Comment by michalschott** [comment]
Updated

**Pull request merged**